packages/x-archetype-utils/src/home/home-template.html

```diff
@@ -0,0 +1,1425 @@
+              width="56"
+              height="33"
+            >
+              <path
+                d="M285.982 236.439L340.36 266.978"
+                stroke="#D3D3D3"
+                stroke-width="1.79431"
+                stroke-dasharray="4.74 4.74"
+              />
+            </mask>
+            <g mask="url(#mask11_5565_97883)">
+              <path
+                d="M285.982 236.439L340.36 266.978"
+                stroke="#D1D1D1"
+                stroke-width="1.79431"
+                stroke-dasharray="4.74 4.74"
+              />
+            </g>
+            <mask
+              id="mask12_5565_97883"
+              style="mask-type: alpha"
+              maskUnits="userSpaceOnUse"
+              x="273"
+              y="221"
+              width="101"
+              height="58"
+            >
+              <path
+                d="M273.623 222.131L372.572 277.351"
+                stroke="#D3D3D3"
+                stroke-width="1.79431"
+                stroke-dasharray="4.74 4.74"
+              />
+            </mask>
+            <g mask="url(#mask12_5565_97883)">
+              <path
+                d="M273.623 222.131L372.572 277.351"
+                stroke="#D1D1D1"
+                stroke-width="1.79431"
+                stroke-dasharray="4.74 4.74"
+              />
+            </g>
+          </svg>
+        </div>
+        <div class="x-archetype-circle"></div>
+      </div>
+      <div class="x-archetype-divider"></div>
+    </div>
+  </section>
+  <section class="x-archetype-section2 x-archetype-row">
+    <div
+      class="
+        x-archetype-section2__container
+        x-archetype-list x-archetype-align-center
+        x-archetype-gap--09
+        x-archetype-row__item
+      "
+    >
+      <div class="x-archetype-margin-top--08 x-archetype-margin-bottom--08">
+        <!--      TODO: Replace by GIF url-->
+        <img
+          src="https://docs.empathy.co/assets/overview-product-results-card.ea7aa783.svg"
+          width="100%"
+        />
+      </div>
+      <div class="x-archetype-list">
+        <p class="x-archetype-section2__title">
+          Know how to integrate the Archetype in your website.
+        </p>
+        <p class="x-archetype-section2__subtitle">
+          Please have a look to the
+          <a
+            class="x-archetype-section2__link"
+            target="_blank"
+            href="https://docs.empathy.co/develop-empathy-platform/build-search-ui/web-archetype-integration-guide.html"
+          >
+            integration documentation
+          </a>
+          . Just load the Archetype JS file and initialise it!
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="x-archetype-section3 x-archetype-row">
+    <div class="x-archetype-section3__container">
+      <div class="x-archetype-section3__list x-archetype-list x-archetype-gap--06">
+        <div class="x-archetype-list x-archetype-align-center x-archetype-gap--05">
+          <svg
```

Comment:
This SVG with the circle should be paired with` Built on open-source technology` element, it was wrong in design.

![image](https://user-images.githubusercontent.com/4663897/211547404-56e70723-3251-4683-859a-59876de7a09f.png)